include/service/entities/article_service.js

```diff
@@ -191,6 +191,28 @@ ArticleService.prototype.find = function(where, options, cb) {
 };
 
 /**
+ * Updates articles
+ *
+ * @param {String} articleId	id of article
+ * @param {Object} options	
+ * @param {Object} fields	fields to update	
+ * @param {Function} cb      Callback function
+ */
+ArticleService.prototype.update = function(articleId, options, fields, cb) {
+	var where = {
+		_id: articleId
+	}
+	
+	var dao  = new pb.DAO();
+	dao.updateFields('article', where, fields, options, function(err, result) {
+	   	if (util.isError(err)) {
+			return cb(err, false);
+		}
+	   	return cb(null, result)
```

Comment:
Since you aren't changing the result it is safe to pass the cb directly to ```dao.updateFields``` such that your code would look like:
```
dao.updateFields(this.getContentType(), where, fields, options, cb);
```